### PR TITLE
5781: Attempt to fix disappearing scrollbar

### LINKED ISSUE #5781: Disappearing scrollbar in diff control
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**
Horizontal scrollbar disappears even though there is hidden content
image

or
(notice that you can't scroll to the right in diff)
image

The most annoying cases are where no amount of vertical scrolling reveals horizontal scrollbar. Then the only way to view a diff is with an external tool.

**Environment you encounter the issue:**

- GitExtensions version: master @ 61b97e7d71f9ae067774674afe451c109ea8851d

**Did this work in previous version of GitExtensions (which)?**
Yes, 2.5 or probably before the diff/text viewer control was refactored to show scrollbars on per-need basis

### PR DESCRIPTION
**Pull request by spdr870**
Fixes #5781 

Changes proposed in this pull request:
- Refresh texteditor more often. It will hurt performance a little, but will maybe fix the scrollbar issue.

What did I do to test?
- Manual testing (click files, move splitters, observer scrollbars, repeat)

### CODE REVIEW

File: GitUI/Editor/FileViewerInternal.cs [hunk 1]
```diff
@@ -131,15 +131,12 @@ public void SetText(string text, Action openWithDifftool, bool isDiff = false)
...
-            TextEditor.BeginUpdate();
-
             TextEditor.Text = text;
 
             // important to set after the text was changed
             // otherwise the may be rendering artifacts as noted in #5568
             TextEditor.ShowLineNumbers = !isDiff;
-
-            TextEditor.EndUpdate();
+            TextEditor.Refresh();
```
Comment (RussKie) [inline GitUI/Editor/FileViewerInternal.cs]: IIRC, without Being/EndUpdate the control would badly flicker...
I'm going to test it

Comment (RussKie) [reply GitUI/Editor/FileViewerInternal.cs]: Here's a flicker example:

flicker

Comment (spdr870) [reply GitUI/Editor/FileViewerInternal.cs]: How about now? I put the beginupdate back. It still fixes the problem and the flicker is gone.

Comment (spdr870) [reply GitUI/Editor/FileViewerInternal.cs]: Ignore my previous comment.... with the beginupdate, the but is back...:(

Comment (spdr870) [reply GitUI/Editor/FileViewerInternal.cs]: Ok, this one helps a little for reducing the flicker! I just put the beginupdate back, and put the refresh outside the update block.

**Pull request merged**